frontend/src/component/context/ContectFormChip/ContextFormChip.tsx

```diff
@@ -1,34 +1,69 @@
-import { useStyles } from 'component/context/ContectFormChip/ContextFormChip.styles';
 import { Cancel } from '@mui/icons-material';
 import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { styled } from '@mui/material';
 
 interface IContextFormChipProps {
     label: string;
     description?: string;
     onRemove: () => void;
 }
 
+const StyledContainer = styled('li')(({ theme }) => ({
+    display: 'grid',
+    lineHeight: 1.25,
+    gridTemplateColumns: '1fr auto',
+    alignSelf: 'start',
+    alignItems: 'start',
+    gap: theme.spacing(1),
+    padding: theme.spacing(1),
+    background: theme.palette.secondaryContainer,
+    borderRadius: theme.shape.borderRadius,
+    wordBreak: 'break-word',
+}));
+
+const StyledLabel = styled('div')(({ theme }) => ({
+    fontSize: theme.fontSizes.smallBody,
+}));
+
+const StyledDescription = styled('div')(({ theme }) => ({
+    fontSize: theme.fontSizes.smallerBody,
+    color: theme.palette.neutral.main,
+}));
+
+const StyledButton = styled('button')(({ theme }) => ({
+    all: 'unset',
+    lineHeight: 0.1,
```

Comment:
This value looks like a hack

Reply:
I know. However in this PR I didn't intend to resolve all the hacks. It was mostly styled components and replacing color/spacing that had obvious values.